### DISCUSSION TITLE
Create a repo of scripts

### DISCUSSION THREAD

**Discussion by axodys** [johnlindquist/kit#469] [Errors]
I wanted to try creating a new .kenv to hold the scripts that I would like to sync between the different macs I use, but it doesn't look like that functionality is fully working right now. I think the original script to create a new .kenv was under the Help menu based on at least one video I found, but it looks like at some point it was moved under the Kit tab and renamed. If I go under "Kit > Create a repo of scripts" I'm able to choose a name and it shows what the path of the new .kenv will be, but hitting the return key is ignored and the field remains editable so a new .kenv is never created. I'm using version (1.30-beta.64) at the moment and have seen this behavior on all my macs. Am I doing something wrong or is there a bug here?

**Comment by johnlindquist** [comment]
It's a bug, I'll get it sorted out tomorrow. Thanks for reporting!

**Reply by johnlindquist** [reply]
This should be fixed in the build from this past weekend. (Forgot to post here until now)
